src/Finder/ClassNamesFinder.php

```diff
@@ -32,7 +32,9 @@ public function resolveClassNamesToCheck(array $filePaths): array
             $filesWithClasses[] = new FileWithClass(
                 $filePath,
                 $classNames->getClassName(),
-                $classNames->hasParentClassOrInterface()
+                $classNames->hasParentClassOrInterface(),
+                $classNames->getAttributes(),
+                $classNames->getAttributesByMethod(),
```

Comment:
I wonder, how is the method name relevant here? We only target classes.
Could me more clear to check if there is an attribute anywhere in the class.

Reply:
I thought it could be helpful for people that use `--json` to further exclude attributes on specific method names. 

But if you don't like it, we can also put the class + method attributes in a flat unique array.

Reply:
> But if you don't like it, we can also put the class + method attributes in a flat unique array.

The simpler, the better :+1:

Reply:
Done!